.github/workflows/linters.yml

```diff
@@ -18,20 +18,20 @@ jobs:
         run: npm install -g @lhci/cli@0.11.x
       - name: Lighthouse Report
         run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: "18.x"
-      - name: Setup Webhint
-        run: |
-          npm install --save-dev hint@7.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
-      - name: Webhint Report
-        run: npx hint .
```

Comment:
Here is a possible code review comment to tell a student that editing of a linters file is a serious violation of the Code review guideline and their project will be invalidated next time:

---

Hello, thank you for submitting your project. I have noticed that you have edited the linters file in your pull request. This is a serious violation of the Code review guideline and it is not acceptable. The linters file is provided by Microverse to ensure that your code follows the best practices and standards. Editing the linters file can compromise the quality and integrity of your code and your project. Therefore, you should never edit the linters file in any way.

I have decided to not invalidate your project this time because the first reviewer already gave you a review on it and this is a rereview on the same pull request. However, this is a one-time exception and you should not expect it to happen again. If you edit the linters file in any future project, your project will be invalidated and you will have to resubmit it with the original linters file. This will delay your progress and affect your performance.

Please restore the original linters file in your pull request and make sure that your code passes all the linter checks without any errors or warnings with the exception of webhint . This will ensure that your code is clean, consistent and professional. If you have any questions or need any help, feel free to ask. Good luck! 😊

We are aware that most of you are experiencing Webhint issues, we are currently working on a solution👨‍💻. For now, please go ahead and request for review if you only have the Webhint error on your PR. Our CRs would be happy to review your PRs 🙂